docs/architecture-decisions/ls-bridge-graceful-shutdown.md

```
@@ -136,103 +138,38 @@ abandoned, and its tracked requests fail exactly like pending responses.
-// Shutdown sequence continues
-async fn graceful_shutdown(&self) {
-    // NOW safe: `stdin` is the handle Phase 2 received from the writer —
-    // ownership, not just quiescence
-    self.send_shutdown_request(&mut stdin).await?;
-
-    // Wait for shutdown response...
-    // Send exit notification...
-    // Kill process...
-}
+**Sequence:**
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)